pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -72,10 +72,28 @@ func InitCnsOperator(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavo
 
 	var volumeManager volumes.Manager
 	if clusterFlavor == cnstypes.CnsClusterFlavorWorkload || clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
-		vCenter, err := cnsvsphere.GetVirtualCenterInstance(ctx, cnsOperator.configInfo, false)
-		if err != nil {
-			return err
+		var (
+			vCenter  *cnsvsphere.VirtualCenter
+			err      error
+			vcconfig *cnsvsphere.VirtualCenterConfig
+		)
+		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.MultiVCenterCSITopology) {
+			vcconfig, err = cnsvsphere.GetVirtualCenterConfig(ctx, cnsOperator.configInfo.Cfg)
```

Comment:
why you are only getting one vCenter Config when `MultiVCenterCSITopology` is enabled?